quick_setup_samples/quck_setup_for_macos_x64.md

```diff
@@ -0,0 +1,68 @@
+# Quick Setup for macos x64
+
+In case you would like to try a fool-proof setup for macos from scratch, you can try the following steps.
+You can also skip some of the steps if it is not applicable, for example, you don't need to install Java 1.8 if you have it already installed.
+
+You can also try your own variations. But the example provided here makes it easy for anybody to setup a powpeg node from scratch in an easy way.
+
+## Steps
+
+1. Install Java OpenJDK 1.8 from the binaries (if it is not installed yet)
+    1. You can download it from here: `https://github.com/AdoptOpenJDK/openjdk8-binaries/releases/download/jdk8u-2021-05-08-08-14/OpenJDK8U-jdk_x64_mac_hotspot_2021-05-08-08-14.tar.gz`
+        1. `cd` to the directory where you want to download it. It could be like `cd /Library/Java/JavaVirtualMachines/`
+        2. Download it like this: `sudo curl -o adoptopenjdk-8.tar.gz -L https://github.com/AdoptOpenJDK/openjdk8-binaries/releases/download/jdk8u-2021-05-08-08-14/OpenJDK8U-jdk_x64_mac_hotspot_2021-05-08-08-14.tar.gz`
+        3. Unzip it like this: `sudo tar -zxvf adoptopenjdk-8.tar.gz`
+        4. The unzipped directory could be named like this `jdk8u302-b01`. Run an `ls` command to see it.
+    2. Add it to the `.bash_profile`.
```

Comment:
Since the last two macOS versions, the default shell in macOS has been zsh(https://support.apple.com/lv-lv/guide/terminal/trml113/2.14/mac/14.0). Therefore, I think this should be done in the zsh profile file, and the same applies to the rest of the configuration done in the bash env files.